enterprise-suite/scripts/build-operator.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+console_version="1.2.0"
+docker_registry="lightbend-docker-commercial-registry.bintray.io"
+image_name="enterprise-suite/console-operator"
+full_docker_name="${docker_registry}/${image_name}:${console_version}"
+
+operator-sdk new console-operator --type=helm --kind=LBConsole \
+    --api-version=lightbend.com/v1alpha1 --helm-chart-repo=https://repo.lightbend.com/helm-charts \
+    --helm-chart=enterprise-suite --helm-chart-version=${console_version}
+
+cd console-operator
+
+operator-sdk build ${full_docker_name}
+
+docker login -u ${DOCKER_USERNAME} -p ${DOCKER_PASSWORD} ${docker_registry}
+docker push ${full_docker_name}
```

Comment:
We don't want to push the image on every commit - it should happen on tags only. So need to split this into build and release as separate jobs I think.

Reply:
Sure, wasn't planning to. This is just a temporary WIP setup, I'll make sure it builds/pushes on tagged master commits.

I'm thinking we don't need to separate push and build as long as both happen only on version tags.

Reply:
Ok, changed it now. Not exactly separate build and push, but here's what happens:

- On branch commits it builds & compares diff of generated and commited k8s resources files so they don't get out of sync.
- On tagged commits it builds and then publishes docker image as a final step in the build script.